### DISCUSSION TITLE
Mobile responsiveness for our application

### DISCUSSION THREAD

**Discussion by thithuongmai** [thithuongmai/T24S2T1EXTG05#7] [General]
Should we implement different layouts or input methods (e.g., sliders for interest rate and loan term) to enhance usability on mobile platforms?

**Comment by thomasMatson** [comment]
Sliders are a great way to enhance the user experience on mobile platforms. I think we should aim to have consistency across all mediums. I suggest we perform a survey on user opinions for different input methods, and choose the most popular ones.
